packages/kbn-doc-links/src/types.ts

```diff
@@ -108,9 +108,15 @@ export interface DocLinks {
     readonly bulkApi: string;
     readonly configuration: string;
     readonly connectors: string;
+    readonly connectorsMongoDB: string;
+    readonly connectorsMySQL: string;
+    readonly connectorsWorkplaceSearch: string;
+    readonly contentExtraction: string;
     readonly crawlerGettingStarted: string;
     readonly crawlerManaging: string;
     readonly crawlerOverview: string;
+    readonly documentLevelSecurity: string;
+    readonly ingestPipelines: string;
```

Comment:
Do you need to add `secureCluster` or `elasticsearchSecureCluster`?

Everything else looks to be consistent with `packages/kbn-doc-links/src/get_doc_links.ts` and `x-pack/plugins/enterprise_search/public/applications/shared/doc_links/doc_links.ts`

Reply:
the `elasticsearch` top level object [doesn't list out its docLinks individually](https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/types.ts#L246) (presumambly there are many)